src/nvim/api/vim.c

```diff
@@ -552,6 +552,18 @@ Integer vim_name_to_color(String name)
   return name_to_color((uint8_t *)name.data);
 }
 
+Dictionary vim_get_color_table(void)
```

Comment:
`vim_get_color_map` ? We should probably stick to consistently referring to dictionaries/maps for the public API at least. No need to introduce another synonym "table".